db/schema.rb

```diff
@@ -10,23 +10,23 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_08_20_124107) do
+ActiveRecord::Schema.define(version: 2021_08_23_081824) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "actions", force: :cascade do |t|
-    t.string "name"
-    t.string "description"
-    t.integer "points"
-    t.string "status"
-    t.boolean "repeatable"
-    t.boolean "action_of_the_month"
-    t.bigint "category_id", null: false
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
-    t.index ["category_id"], name: "index_actions_on_category_id"
-  end
+  # create_table "actions", force: :cascade do |t|
+  #   t.string "name"
+  #   t.string "description"
+  #   t.integer "points"
+  #   t.string "status"
+  #   t.boolean "repeatable"
+  #   t.boolean "action_of_the_month"
+  #   t.bigint "category_id", null: false
+  #   t.datetime "created_at", precision: 6, null: false
+  #   t.datetime "updated_at", precision: 6, null: false
+  #   t.index ["category_id"], name: "index_actions_on_category_id"
+  # end
```

Comment:
Why is this section commented out? Wont we need it or will we need it later? :)

Reply:
If I remember correctly you don't touch the schema file, it manages itself when executing rails-code. So these changes might be automatically generated, @netlight-saraandersson ?